### PR TITLE
homework 3

### PR DESCRIPTION
**Pull request by Tobu9009**

### CODE REVIEW

File: test/specs/Class3/Practice.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+describe('Verify darksky', () =>{
+    it('Verify we can change temp unit in darksky ', async() =>{
+        await browser.url('https://www.darksky.net')
+        await browser.pause(2000);
+        await $('//div[@id="header"]//div[contains(@class, "selectric-units")]//div[@class="selectric"]//b').click();
+        await browser.pause(2000);
+        await $('//div[contains(@class, "selectric-open")]//li[contains(text(), "˚C") and contains(text(), "mph")]').click();
+        await browser.pause(2000);
+        const newCelsiusTemp = await $('//div[@id="header"]//div[contains(@class,"selectric-units")]//div[@class="selectric"]//span').getText();
+        expect(`\n\ntempUnit -> ${tempUnit}`);
```
Comment (ahsansiddiki) [inline test/specs/Class3/Practice.js:18]: This isn't a great verification, can be done better.

File: test/specs/Class3/WebElements3.js [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+        const loginEmail = await $('//input[@name="email"]')
+        await loginEmail.setValue("abcd@1234.com")
+        await browser.pause(2000)
+
+        const loginPwd = await $('//input[@data-testid="royal_pass"]')
+        await loginPwd.setValue("abcd1234")
+        const loginButton = await $('//button[text()="Log In"]')
+        await loginButton.click();
+        await browser.pause(2000)
+
```
Comment (ahsansiddiki) [inline test/specs/Class3/WebElements3.js:15]: Verification missing.

File: test/specs/Class4/Homework3.js [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+        for (const autoSuggestionElement of autoSuggestionElements) {
+            const suggestionText = await autoSuggestionElement.getText();
+            if (suggestionText.toLowerCase().localeCompare('manila'.toLowerCase()) === 0) {
+                await autoSuggestionElement.click();
+                break;
+            }
+        }
+        // 4. Select Aug 20 to Sep-5 as check-in and check-out respectively
+        await $('#date_form_field-btn').click();
+        const allAugDateElements = await $$('//h2[text()="August 2022"]/following-sibling::table//button[not(@disabled)]');
```
Comment (ahsansiddiki) [inline test/specs/Class4/Homework3.js:56]: `August 2022` This is hard coding, this selector not gonna work after Aug.

**Review by ahsansiddiki — APPROVED** [review]

**Comment by ahsansiddiki** [comment]
Left some improvement suggestion for future.

**Comment by Tobu9009** [comment]
This one was just notes that I took from deepak.

**Comment by Tobu9009** [comment]
This code was also taken from the notes from deepak

**Pull request merged**